packages/create-sitecore-jss/src/templates/nextjs/src/Layout.tsx

```diff
@@ -15,14 +14,14 @@ import Navigation from 'src/Navigation';
 const publicUrl = getPublicUrl();
 
 interface LayoutProps {
-  sitecoreContext: SitecoreContextValue;
+  layoutData: LayoutServiceData;
 }
 
-const Layout = ({ sitecoreContext: { route } }: LayoutProps): JSX.Element => {
+const Layout = ({ layoutData }: LayoutProps): JSX.Element => {
   return (
     <>
       <Head>
-        <title>{route?.fields?.pageTitle?.value || 'Page'}</title>
+        <title>{layoutData.sitecore.route?.fields?.pageTitle?.value || 'Page'}</title>
```

Comment:
Minor, but do we want to destructure the `route` prop as we were before (more concise)?

Reply:
Done